services/api/mw2pdf/package.json

```diff
@@ -2,11 +2,11 @@
   "name": "mw2pdf",
   "version": "0.0.1",
   "description": "Given mediawiki URL, output PDF",
-  "main": "index.js",
+  "private": true,
```

Comment:
We (or maybe just I) do hope to eventually publish mw2pdf as a standalone package.

We haven't done that yet, so I'm fine with this line though I admit I don't know what it gets us aside from protection from accidental publishing.

Reply:
I agree that we do ultimately want to publish as a package, but I think that prevention from accidental publishing is *exactly* what it gets you -- a Good Thing (tm) because I am error prone. If you want me to remove this line, that is perfectly okay. Just let me know your preference. :-)

Reply:
Let's keep it!  We can remove it if / when we're ready to publish.

Reply:
Sounds good!